packages/addon-dev/src/rollup-gjs-plugin.ts

```diff
@@ -14,11 +13,10 @@ export default function rollupGjsPlugin(
   return {
     name: PLUGIN_NAME,
 
-    load(id: string) {
+    transform(input: string, id: string) {
```

Comment:
@ef4 I noticed that now the gjs plugin must come before the babel plugin in the rollup config. That would make this a breaking change